src/utils/components/CPUDescription/utils/utils.ts

```diff
@@ -1,8 +1,17 @@
+import { V1CPU, V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
 import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getName, getNamespace } from '@kubevirt-utils/resources/shared';
+import useVMI from '@kubevirt-utils/resources/vm/hooks/useVMI';
 
 export const CpuMemHelperTextResources = {
   ActualVM: t('This VirtualMachine has'),
   FutureVM: t('The VirtualMachine will have'),
   Template: t('The template has'),
   VMI: t('The VirtualMachineInstance has'),
 };
+
+export const useCPUData = (vm: V1VirtualMachine): V1CPU => {
```

Comment:
please expand this hook and move it to its own file

Reply:
Yes, thanks, makes sense.